api/server.go

```diff
@@ -112,11 +143,23 @@ func NewServer(ctx context.Context, conf Config) (*Server, error) {
 		}
 	}()
 
+	errc := make(chan error)
+	go func() {
+		errc <- s.rpcWebProxy.ListenAndServe()
```

Comment:
It doesn't look like you're reading the error channel. In any case, you can just do a normal error check... like @jsign mentioned on the fil-tools PR. You'll just have to check for and ignore the server closed error.